component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -586,6 +586,7 @@ private void proceedWithOTP(HttpServletResponse response, AuthenticationContext
         String tenantAwareUsername = MultitenantUtils.getTenantAwareUsername(username);
         UserRealm userRealm = SMSOTPUtils.getUserRealm(tenantDomain);
         int tokenLength = SMSOTPConstants.NUMBER_DIGIT;
+        long tokenExpiryTime = SMSOTPConstants.VALIDITY_TIME;
```

Comment:
Can we use the term OTP instead of the token? Is there a particular reason not to do so?

Reply:
Fixed